doc/analyze-from-tfs.md

```diff
@@ -140,6 +140,9 @@ Try modifying the build definition to remove the SonarQube.MSBuild.Runner.exe en
 
 Most analysis-related configuration or execution errors will cause the build to fail and will be appear on the Build Summary. Additional information can be found by viewing the logs or diagnostic information (i.e. by clicking on **View Log**, or **Diagnostics** at the top of the Build Summary page).
 
+#### During a XAML build, the analysis fails with a "System.ArgumentNullException ... Parameter name: tfsUri"
+
+Make sure you are using at least TFS 2013 Update 2 agent software. 

```

Comment:
Following the instructions in the doc (i.e. "Search for Team Foundation Server Object Model") finds three versions for 2013: the one you listed and ones for updates 4 and 5. However, if it works with the lowest update version then it should be fine.